src/components/Footer.js

```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 import styles from "./Footer.module.css";
 
-export function Footer() {
+export default function Footer() {
```

Comment:
There's no need to change this.

```suggestion
export function Footer() {
```